src/libaktualizr/storage/storage_common_test.cc

```diff
@@ -384,24 +384,45 @@ TEST(storage, load_store_installed_versions) {
   }
 }
 
-/* Load and store an installation result. */
-TEST(storage, load_store_installation_result) {
+/* Load and store a installation results (ecu and device). */
+TEST(storage, load_store_installation_results) {
```

Comment:
If you change this line, you should probably change the corresponding line in `actions.md`. And actually, I think you've added some new actions that are worth specifying. (And very minor, but you can drop the "a".)

Reply:
Sorry to be stubborn, but in lieu of a better way to link the test case to the action, I think the comment here should match the actions exactly, e.g.:
```
/*
 * Load and store an ecu installation result in an SQL database
 * Load and store a device installation result in an SQL database
 */
```

Reply:
Sure thing. Must have thought that the "SQL database" part is too specific, but I'll change it for consistency.

Reply:
Fair point. The specificity in a place that should be generic is annoying, but I think the need for linkage outweighs that at least for now.